### ISSUE TITLE
Formatting time with input in milliseconds

### ISSUE THREAD

**Issue by ElectricCoffee** [adamwdraper/Numeral-js#292]
It appears that it's not currently possible to format the time given an input in milliseconds as of yet, either in the format `hh:MM:ss:mmm` or simply `hh:MM:ss` where `MM` is minutes and `mmm` is milliseconds.
Or you can and I just couldn't find it in the docs

**Comment by jamiter** [comment]
Hi, this repository is no longer maintained. You can get an improved version here: https://github.com/foretagsplatsen/numbro. That one is still maintained and you can report your issues there.

**Comment by ElectricCoffee** [comment]
Would be nice if it was apparent for others that the library isn't maintained. Nothing on the front page or readme says so

**Comment by jamiter** [comment]
Agreed. It's not something the maintainer officially announced though, but he hasn't changed anything in 2 years and doesn't respond to tickets. That's why the fork was created, which has been actively developed and maintained since.

**Comment by ElectricCoffee** [comment]
Alright, thanks!

**Comment by brunoporto** [comment]
https://github.com/adamwdraper/Numeral-js/pull/614

format: 00:00:00.0

**Comment by vincent-ogury** [comment]
any news for this patch?

I'm also looking forward to time with ms.
